<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface_2</class>
 <widget class="QMainWindow" name="interface_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>349</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Interface Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>IP Serveur</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditIPServeur">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>172.18.58.x</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxPort">
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>8888</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Texte à envoyer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="TexteAEnv">
              <property name="placeholderText">
               <string>Texte</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="BoutonEnvoyer">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Envoyer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="Connexion">
          <property name="text">
           <string>Connexion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BoutonQuitter">
          <property name="text">
           <string>Quitter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Message et état</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="TexteEtEtat"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BoutonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>interface_2</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>771</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
